iina/uk.lproj/MainMenu.strings

```diff
@@ -403,7 +403,7 @@
 "mGw-jm-5x2.title" = "Другі субтитри";
 
 /* Class = "NSMenuItem"; title = "Enter Picture in Picture"; ObjectID = "mND-N8-z6o"; */
-"mND-N8-z6o.title" = "Режим «Кадр у кадрі»";
+"mND-N8-z6o.title" = "Режим «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">